packages/workbox-background-sync/test/browser/request-queue.js

```diff
@@ -30,6 +30,14 @@ describe('request-queue tests', () => {
     callbacks,
   });
 
+  const getDelayedPromise = () => {
+    return new Promise((resolve)=>{
+      setTimeout(()=>{
```

Comment:
Please use consistent spacing for arrow functions here and in line 34. There should be a single whitespace character around the arrow, e.g. `() => {`.